specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/getOperations.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.Subscription/createSubscription/action",
+            "display": {
+              "provider": "Microsoft Subscription",
+              "resource": "Resources",
+              "operation": "Create a subscription",
+              "description": "Create an Azure subscription."
```

Comment:
The description isn't part of the schema and is causing validation errors, see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/455086153).